sklearn/impute/tests/test_impute.py

```diff
@@ -1516,6 +1518,9 @@ def test_simple_imputation_inverse_transform_exceptions(missing_value):
         ),
         ("a", ["min_value", "min_valuevalue"], object, "a", 2),
         ("min_value", ["min_value", "min_value", "value"], object, "z", 2),
+        # array of uncomparable object dtype
+        (min(['a', None, 0], key=lambda x: murmurhash3_32(pickle.dumps(x))),
+          ['a', None], object, 0, 1),
```

Comment:
Let's add a dedicated non regression test for ties. This one is more for extra and repeat.
Also, let's hardcode the expected result